src/Stats/TransList.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { ListItem } from 'react-native-elements';
+import GradientIcon from '../Common/icon';
+import styles from '../Common/themeStyle';
+import utils from './utils';
+
+export default function TransList(props) {
+  const { transactions } = props;
+
+  return (
+    transactions.map((value, index) => (
+      <View style={styles.card} key={`DateGroup:${index + 1}`}>
+        <View style={styles.cardAlign}>
+          <Text style={styles.cardHeader}>
+            {value[0].date}
```

Comment:
is it an array? Then is definitely should have the name `value`, neither `values`. Please let the reader know what the given variable is.

Reply:
Done, changed it to `groupedRecordsByDate`